### ISSUE TITLE
Expose ClientVpcIpAddress for built Kafka clusters

### ISSUE THREAD

**Issue by KushalP** [hashicorp/terraform-provider-aws#11085]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

When we've created a Kafka cluster, expose the IPs of the individual brokers as an attribute so that they can be used for other configuration. This is already being exposed by the AWS Go SDK: https://github.com/aws/aws-sdk-go/blob/363ee75f96be07cb7dd905ad9fda37d0eeafeb68/service/kafka/api.go#L2204

### New or Affected Resource(s)

* [aws_msk_cluster](https://www.terraform.io/docs/providers/aws/r/msk_cluster.html)

**Comment by carl34** [comment]
Hi, here's some candidate code to support list_nodes with a new output containing a comma separated list of all ClientVpcIpAddress values for a cluster.

https://github.com/carl34/terraform-provider-aws/blob/msk_list_nodes/website/docs/r/msk_cluster.html.markdown

Example:
```
output "broker_client_vpc_ip_address" {
  description = "Broker IP Addresses"
  value       " "${aws_msk_cluster.example.broker_client_vpc_ip_addresses}"
}

broker_client_vpc_ip_addresses = 10.0.0.1,10.0.0.2,10.0.0.3
```


I've tested the code successfully, I'm still working on unit testing. Please let me know if someone who is familiar with the MSK unit tests can assist.

**Comment by carl34** [comment]
[WIP] PR created, https://github.com/terraform-providers/terraform-provider-aws/pull/12821

**Comment by carl34** [comment]
Unit test output added to https://github.com/terraform-providers/terraform-provider-aws/pull/12821

**Comment by ewbankkit** [comment]
@KushalP Thanks for the contribution :tada: :clap:.
I think the best solution for this will be a new `aws_msk_nodes` data source as implemented in #20615.
